spec/features/jobs/user_edits_a_job_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe "user edits an existing job" do
+  scenario "a user can edit a job" do
+    company = Company.create!(name: "StackOverflow")
+    job = company.jobs.create!(title: "Rails Developer",
+      description: "do rails things", level_of_interest: 3, city: "Denver")
+
+    visit edit_company_job_path({id: job.id, company_id: company.id})
```

Comment:
Might take out the curly braces here. Like that you know this is a hash, but feels a little more idiomatic without them.